### PR TITLE
Update jdl.md

### PR DESCRIPTION
**Pull request by deepu105**
Making the doc clearer after https://stackoverflow.com/questions/44634055/jhipster-jdl-not-generating-anything-because-the-database-type-must-be-passed

### CODE REVIEW

File: pages/jdl.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ The JDL is a JHipster specific domain language where we have added the possibili
...
 
-You can generate entities from a JDL file using the `import-jdl` sub-generator, by running `jhipster import-jdl your-jdl-file.jh`.
-You can also generate entities from a JDL file using [JHipster UML]({{ site.url }}/jhipster-uml/), by running `jhipster-uml your-jdl-file.jh` from the root of the generated JHipster application. To learn more about JHipster UML, and install it, go to the [JHipster UML documentation]({{ site.url }}/jhipster-uml/).
+Once you have a generated project (either existing one or generated with `yo jhipter`), you can generate entities from a JDL file using the `import-jdl` sub-generator, by running `jhipster import-jdl your-jdl-file.jh` (make sure to execute this command under your JHipster project). You can also generate entities from a JDL file using [JHipster UML]({{ site.url }}/jhipster-uml/), by running `jhipster-uml your-jdl-file.jh` from the root of the generated JHipster application. To learn more about JHipster UML, and install it, go to the [JHipster UML documentation]({{ site.url }}/jhipster-uml/).
```
Comment (deepu105) [inline pages/jdl.md]: We now use the CLI and not yo so this should be just `jhipster`
Btw I think you forgot to open the PR so I did it

Comment (jdubois) [inline pages/jdl.md]: There is still a 'yo jhipster'

Comment (deepu105) [inline pages/jdl.md]: I think we can merge and correct it

Comment (jdubois) [inline pages/jdl.md]: I can't, I'm on my phone and travelling to a place with poor network access

Comment (jdubois) [inline pages/jdl.md]: Otherwise I'll do this on Mo'day

**Comment by deepu105** [comment]
ok, ill do it then


Thanks & Regards,
Deepu

**Pull request merged**